airflow/ui/src/pages/Dag/Code/Code.tsx

```diff
@@ -72,12 +72,25 @@ export const Code = () => {
   const defaultWrap = Boolean(useConfig("default_wrap"));
 
   const [wrap, setWrap] = useState(defaultWrap);
+  const [copyStatus, setCopyStatus] = useState("Copy Code");
 
   const toggleWrap = () => setWrap(!wrap);
   const { colorMode } = useColorMode();
 
   const style = colorMode === "dark" ? oneDark : oneLight;
 
+  const copyCode = async () => {
```

Comment:
There is already a `clipboard` component from Chakra available in `src/components/ui/Clipboard.tsx` which can be used here since it's used across the application.